*** testsEF/A-009-034-M-ok.out	2016-12-18 19:30:04.191430527 +0000
--- testsEF/A-009-034-M-ok.outhyp	2016-12-18 19:55:59.563774269 +0000
***************
*** 27,40 ****
  Escolha uma opção: Testar partilha de variáveis
  antes do call
  10
- 11 14
- depois do primeiro call
- 11
  14
! 12 15
  depois do segundo call
! 12
! 15
  fim
  Menu de Edição
  1 - Listar programa
--- 27,40 ----
  Escolha uma opção: Testar partilha de variáveis 
  antes do call 
  10 
  1   4 
! depois do primeiro call 
! 10 
! 0 
! 2   5 
  depois do segundo call 
! 10 
! 0 
  fim 
  Menu de Edição
  1 - Listar programa
