*** testsEF/A-012-008-M-ok.out	2016-12-18 18:41:37.596547888 +0000
--- testsEF/A-012-008-M-ok.outhyp	2016-12-18 18:51:09.603636788 +0000
***************
*** 28,33 ****
--- 28,35 ----
  d
  g
  x
+ xx
+ y
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
***************
*** 53,59 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: x
  xx
  y
  Menu de Edição
--- 55,64 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: a
! d
! g
! x
  xx
  y
  Menu de Edição
