*** testsEF/A-010-008-M-ok.out	Sat Dec 10 10:37:52 2016
--- testsEF/A-010-008-M-ok.outhyp	Sun Dec 11 10:52:22 2016
***************
*** 28,35 ****
--- 28,38 ----
  c
  d
  g
+ l
  v
  x
+ xx
+ y
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
***************
*** 55,61 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: l
  x
  xx
  y
--- 58,69 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: a
! c
! d
! g
! l
! v
  x
  xx
  y
