*** testsEI/A-002-002-M-ok.out	Wed Dec  7 12:41:06 2016
--- testsEI/A-002-002-M-ok.outhyp	Wed Dec  7 12:43:05 2016
***************
*** 15,26 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (neg 1)
  (add 1 2)
! (sub 1 2)
! (mul 1 2)
  (div 1 2)
! (mod 1 2)
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 15,26 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (null 1)
  (add 1 2)
! (null 1 2)
! (null 1 2)
  (div 1 2)
! (null 1 2)
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
