*** testsEI/A-008-003-M-ok.out	2016-12-07 10:47:18.369551544 +0000
--- testsEI/A-008-003-M-ok.outhyp	2016-12-09 12:02:45.329098468 +0000
***************
*** 7,13 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: O ficheiro não existe.
  Menu Principal
  1 - Criar
  2 - Abrir
--- 7,13 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: O ficheiro 'awerty' não existe.
  Menu Principal
  1 - Criar
  2 - Abrir
