*** testsEI/A-002-005-M-ok.out	Wed Dec  7 12:41:06 2016
--- testsEI/A-002-005-M-ok.outhyp	Wed Dec  7 12:43:05 2016
***************
*** 15,40 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (neg (add 2 3))
  (add 1 (add 20 eee))
! (sub (mul 2 (add 2 3)) 2)
! (mul (mul aa ee) 2)
! (div 1 (sub 2 (sub 3 4)))
! (mod 1 (set a 23))
! (eq (add 2 3) (add 4 1))
! (ne (sub aa 3) 2)
! (lt (sub 2 3) (add 4 1))
! (gt (add 2 3) (add 4 1))
! (le (add 2 3) (add 4 1))
! (ge (add 2 3) (mul aaa 1))
! (not (sub aa 1))
  (and (add 2 3) (add e 1))
! (or (mul 2 3) (sub 4 1))
! (set aa (readi))
! (print "wer" 23 (add a 5) (reads) (readi))
! (seq 1 2 3 4 5 6 7 (add e 6) "asd")
! (if (and aa (add b 2)) (add aa 1) (sub b 1))
! (while (lt aa 10) (set aa (add aa 1)))
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 15,40 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (null (add 2 3))
  (add 1 (add 20 eee))
! (null (null 2 (add 2 3)) 2)
! (null (null aa ee) 2)
! (div 1 (null 2 (null 3 4)))
! (null 1 (null a 23))
! (null (add 2 3) (add 4 1))
! (null (null aa 3) 2)
! (null (null 2 3) (add 4 1))
! (null (add 2 3) (add 4 1))
! (null (add 2 3) (add 4 1))
! (null (add 2 3) (null aaa 1))
! (null (null aa 1))
  (and (add 2 3) (add e 1))
! (null (null 2 3) (null 4 1))
! (null aa (readi))
! (null "wer" 23 (add a 5) (reads) (readi))
! (null 1 2 3 4 5 6 7 (add e 6) "asd")
! (null (and aa (add b 2)) (add aa 1) (null b 1))
! (null (null aa 10) (null aa (add aa 1)))
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
