*** testsEI/A-008-005-M-ok.out	Thu Dec  8 16:52:47 2016
--- testsEI/A-008-005-M-ok.outhyp	Thu Dec  8 16:53:30 2016
***************
*** 7,13 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
--- 7,14 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Erro a guardar o interpretador
! Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
