*** tests/A-009-018-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-009-018-M-ok.outhyp	Tue Dec  6 13:19:48 2016
***************
*** 29,40 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 1
! 1 2
! 123456789
! 012
! exemplo: a + 1 = 1
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 29,35 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
