*** tests/A-002-001-M-ok.out	2016-11-19 11:18:47.718208024 +0000
--- tests/A-002-001-M-ok.outhyp	2016-11-19 19:27:09.356269670 +0000
***************
*** 15,24 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 12
! "abcd"
! "1234"
! aaa
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 15,21 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Sub
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
