*** tests/A-009-008-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-009-008-M-ok.outhyp	Tue Dec  6 13:19:47 2016
***************
*** 15,23 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (print (mod 5 2))
! (print (mod 4 8))
! (print (mod 9 3))
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 15,26 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (print (mod 5 2)
! )
! (print (mod 4 8)
! )
! (print (mod 9 3)
! )
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
***************
*** 26,35 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 1
! 4
! 0
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 29,35 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
