*** tests/A-001-004-M-ok.out	2016-11-18 18:15:32.004870782 +0000
--- tests/A-001-004-M-ok.outhyp	2016-11-18 20:37:08.082985122 +0000
***************
*** 7,14 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: O programa s não existe.
! Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
--- 7,13 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
