*** tests/A-009-011-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-009-011-M-ok.outhyp	Tue Dec  6 13:19:48 2016
***************
*** 30,40 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: true
! 0 1
! false
! 0 0
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 30,36 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
