*** tests/A-004-001-M-ok.out	Thu Nov 17 14:55:35 2016
--- tests/A-004-001-M-ok.outhyp	Fri Nov 18 12:00:52 2016
***************
*** 7,13 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do ficheiro do program: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
--- 7,13 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
***************
*** 24,31 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 12
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 24,30 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
