*** tests/A-002-003-M-ok.out	2016-11-19 11:18:47.718208024 +0000
--- tests/A-002-003-M-ok.outhyp	2016-11-19 12:32:22.795821691 +0000
***************
*** 15,30 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (eq 1 1)
! (ne 1 2)
! (lt 1 2)
! (gt 1 2)
! (le 1 2)
! (ge 1 2)
! (not 1)
! (and 1 0)
! (or 0 aa)
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 15,21 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
