*** tests/A-002-001-M-ok.out	Thu Nov 17 14:55:35 2016
--- tests/A-002-001-M-ok.outhyp	Thu Nov 17 23:12:31 2016
***************
*** 15,25 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 12
! "abcd"
! "1234"
! aaa
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 15,21 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
