*** tests/A-009-025-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-009-025-M-ok.outhyp	Tue Dec  6 13:19:49 2016
***************
*** 15,25 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: iguais
! iguais
! diferentes
! diferentes
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 15,21 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
