*** tests/A-001-004-M-ok.out	Wed Nov  9 22:44:41 2016
--- tests/A-001-004-M-ok.outhyp	Wed Nov 16 13:35:29 2016
***************
*** 7,14 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: O programa s não existe.
! Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
--- 7,13 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
***************
*** 17,20 ****
--- 16,20 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: 
\ No newline at end of file
