*** tests/A-004-001-M-ok.out	Sat Nov 19 23:02:46 2016
--- tests/A-004-001-M-ok.outhyp	Mon Nov 21 10:44:53 2016
***************
*** 24,30 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 12
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
--- 24,30 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 3
  Menu de Edição
  1 - Listar programa
  2 - Executar programa
