*** tests/A-011-002-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-011-002-M-ok.outhyp	Tue Dec  6 13:19:51 2016
***************
*** 15,21 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Posição do programa: Expressão: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 15,21 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
***************
*** 23,32 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
  Escolha uma opção: (print 1)
  (print 2)
  (print 3)
! (print "quatro")
  (print 6)
  (print 7)
  (print 8)
--- 23,41 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
+ Escolha uma opção: Menu de Edição
+ 1 - Listar programa
+ 2 - Executar programa
+ 3 - Adicionar expressão ao programa
+ 4 - Substituir expressão no programa
+ 5 - Mostrar os identificadores presentes no programa
+ 6 - Mostrar os identificadores do programa sem inicialização explícita
+ 0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: (print 1)
  (print 2)
  (print 3)
! (print 4)
  (print 6)
  (print 7)
  (print 8)
***************
*** 39,53 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 1
! 2
! 3
! quatro
! 6
! 7
! 8
! 9
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 48,54 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
